.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+# Dependabot configuration for Gradle dependencies, Docker images and Github Actions.
+
+version: 2
```

Please check with @bittelc how to add private repo scanning

---

You can add this our maven registry to scan for private updates:
```suggestion
version: 2
registries:
  pleo-gradle-github-registry:
    type: maven-repository
    url: 'https://maven.pkg.github.com/pleo-io'
    username: "pleobot"
    password: ${{secrets.PLEO_GH_PACKAGE_REGISTRY_READ_TOKEN}}
```

---

I just realised that we do not need it. This is opensource repo and it has no dependency on pleo's private artifacts